analyses/malawi/flooding_trigger/mwi-glofas-skill.md

```diff
@@ -0,0 +1,231 @@
+```python
+from datetime import datetime
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from scipy.stats import norm, pearsonr
+import numpy as np
+import pandas as pd
+import xarray as xr
+import xskillscore as xs
+from scipy.interpolate import interp1d
+from scipy.stats import rankdata
+import os
+from pathlib import Path
+import sys
+from datetime import timedelta
+
+import read_in_data as rd
+from importlib import reload
+reload(rd)
+
+path_mod = f"{Path(os.path.dirname(os.path.abspath(''))).parents[1]}/"
+sys.path.append(path_mod)
+
+from src.indicators.flooding.config import Config
+
+config = Config()
+mpl.rcParams['figure.dpi'] = 300
+
+PLOT_DIR = config.DATA_DIR / 'processed' / 'mwi' / 'plots' / 'flooding'
+EXPLORE_DIR = config.DATA_DIR / 'exploration' / 'mwi' / 'flooding'
+GLOFAS_VERSION = 3
+STATIONS = ['glofas_1', 'glofas_2']
+SAVE_FIG = True
+```
+
+### Read in the GloFAS data
+
+```python
+da_glofas_reanalysis = {}
+da_glofas_reforecast = {}
+da_glofas_forecast = {}
+da_glofas_forecast_summary = {}
+da_glofas_reforecast_summary = {}
+
+for station in STATIONS: 
+    da_glofas_reanalysis[station] = rd.get_glofas_reanalysis(version=GLOFAS_VERSION, station=station)
+    da_glofas_reforecast[station] = rd.get_glofas_reforecast(version=GLOFAS_VERSION, station=station)
+    da_glofas_forecast[station] = rd.get_glofas_forecast(version=GLOFAS_VERSION, station=station)
+    da_glofas_forecast_summary[station] = rd.get_da_glofas_summary(da_glofas_forecast[station])
+    da_glofas_reforecast_summary[station] = rd.get_da_glofas_summary(da_glofas_reforecast[station])
+```
+
+### Read in the return period thresholds
+
+```python
+df_rps = pd.read_csv(EXPLORE_DIR / 'glofas_rps.csv')
```

Comment:
Btw if you are saving a Pandas dataframe with a meaningless index, if you set "index=False" then you won't have the "Unamed: 0" column anymore